elastic/kibana
# Add total_errors to TaskClaim metrics

Towards: https://github.com/elastic/response-ops-team/issues/185

This PR adds `total_errors` to TaskClaim metrics, so we can create an anomaly detection job for it.

## To verify:
Create an Elasticsearch Query rule.
Throw an error in ES Query rule executor by adding `throw new Error('test error');` [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/task_manager/server/task_claimers/strategy_default.ts#L108)

Open http://localhost:5601/api/task_manager/metrics?reset=false to check metrics.

`total_errors` should be reported for task_claim

```
{"task_claim":{"timestamp":"2024-04-26T14:49:51.526Z","value":{"success":0,"total":9,"total_errors":9,"duration":{"counts":[],"values":[]},"duration_values":[]}}
```
